gem/templates/rwanda/base.html

```diff
@@ -141,11 +141,10 @@
                 {% if sections %}
                 <li class="nav-list__item">
                   {% for section_cat in sections %}
-                    {% get_translation section_cat as section %}
-                    {% if section.title == 'Ibibera Ino' or section.title == 'Events' %}
-                      <a href="{% pageurl section.specific %}" class="nav-list__anchor">
-                       {{section.title}}
-                      </a>
+                    {# {% get_translation section_cat as section %} #}
+                    {% content_is section_cat "Events" as is_event %}
+                    {% if is_event == True %}
+                      <a href="{% pageurl section_cat.specific %}" class="nav-list__anchor">{{section_cat.title}}</a>
```

Comment:
Does this not need to be translated somewhere in this? It looks like you put the `get_translation` tag above here into a comment.